docs/tutorial/resources/BDK14-exercises/basic-dl-query/catalog-v001.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
why is this needed?

Reply:
no idea, copied the whole folder across from the repo - will remove all catalog